.ghal.rules.json

```diff
@@ -0,0 +1,38 @@
+{
+  "labels": {
+    "defaults": [ ":watch: Not Triaged" ],
+    "product": {
+      "(?i).*": "Source - Docs.ms",
+      "(?i)dotnet-csharp$": ":books: Area - C# Guide",

```

Comment:
I think only the first one should be kept (Source - Docs.ms) - all the others don't make sense here